docker/colony-cdapp-dev-env-amplify

```diff
@@ -8,14 +8,20 @@ WORKDIR /colonyCDapp
 # Add dependencies from the host
 ADD package.json /colonyCDapp/package.colonyCDapp.json
 ADD amplify /colonyCDapp/amplify
-ADD .graphqlconfig.yml /colonyCDapp/.graphqlconfig.yml
+ADD codegen.js /colonyCDapp/codegen.js
 
 # Note: these are listed individually so that if they change, they won't affect
 # the build of the other images
 ADD docker/files/amplify/run.sh.base /colonyCDappBackend/run.sh
 ADD docker/files/amplify/local-env-info.json.base /colonyCDapp/amplify/.config/local-env-info.json
 ADD docker/files/amplify/local-aws-info.json.base /colonyCDapp/amplify/.config/local-aws-info.json
 ADD docker/files/amplify/aws-exports.js.base /colonyCDapp/src/aws-exports.js
+ADD docker/files/amplify/package.base.json /colonyCDapp/package.base.json
+ADD docker/files/amplify/dependencies /colonyCDapp/dependencies
+ADD docker/files/amplify/generate-package-json.js /colonyCDapp/generate-package-json.js
+ADD docker/files/amplify/fakeschema.graphql /colonyCDapp/fakeschema.graphql
+ADD docker/files/amplify/.graphqlconfig.yml /colonyCDapp/.graphqlconfig.yml
```

Comment:
Please keep the source file naming similar and add `.base` to the end of their name/extension